engine/src/flutter/lib/web_ui/lib/src/engine/web_paragraph/painter.dart

```diff
@@ -14,6 +14,15 @@ import 'debug.dart';
 import 'layout.dart';
 import 'paragraph.dart';
 
+// TODO(mdebbar): Discuss it: we use this canvas for painting the entire block (entire line)
+// so we need to make sure it's big enough to hold the biggest line.
```

Comment:
We plan to move background and decoration drawing directly on Canvaskit, so this TODO will be irrelevant soon.